client/verta/tests/test_entities.py

```diff
@@ -370,28 +371,26 @@ def test_clone(self, experiment_run):
         expt_run.log_artifact("my-artifact", "README.md")
         expt_run.log_code()
 
-        # set various things in the run
-        new_run_no_art = expt_run.clone()
+        # clone run
+        new_run = expt_run.clone()
+        expt_run._fetch_with_no_cache()
+        new_run._fetch_with_no_cache()
 
-        old_run_msg = expt_run._get_proto_by_id(expt_run._conn, expt_run.id)
-        new_run_no_art_msg = new_run_no_art._get_proto_by_id(new_run_no_art._conn, new_run_no_art.id)
```

Comment:
The ER cloning functionality used to have a user-facing option to skip cloning artifacts. It doesn't anymore, but it appears this test still kept the `new_run_no_art_msg` variable (`no_art` for "no artifacts"). I've replaced it with simply `new_run`.